gamestonk_terminal/stocks/government/gov_controller.py

```diff
@@ -141,97 +128,468 @@ def call_quit(self, _):
+            default=5,
+            help="Number of tickers to get",
+        )
+        parser.add_argument(
+            "-a",
+            "--analysis",
+            action="store",
+            dest="analysis",
+            choices=["total", "upmom", "downmom"],
+            type=str,
+            default="total",
+            help="Analysis to look at contracts.",
+        )
+        try:
+            ns_parser = parse_known_args_and_warn(parser, other_args)
+            if not ns_parser:
+                return
 
-    def call_last_house(self, other_args: List[str]):
-        """Process last_house command"""
-        quiverquant_view.last_government(other_args, "house")
+            quiverquant_view.display_qtr_contracts(
+                analysis=ns_parser.analysis, num=ns_parser.num
+            )
+        except Exception as e:
+            print(e, "\n")
 
-    def call_buy_house(self, other_args: List[str]):
-        """Process buy_house command"""
-        quiverquant_view.buy_government(other_args, "house")
+    def call_top_lobbying(self, other_args: List[str]):
+        """Process top_lobbying command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="top_lobbying",
+            description="Top lobbying. [Source: www.quiverquant.com]",
+        )
+        parser.add_argument(
+            "-n",
+            "--num",
+            action="store",
+            dest="num",
+            type=check_positive,
+            default=10,
+            help="Number to show",
+        )
+        parser.add_argument(
+            "--raw",
+            action="store_true",
+            default=False,
+            dest="raw",
+            help="Print raw data.",
+        )
+        parser.add_argument(
+            "--export",
+            choices=["png", "jpg", "pdf", "svg", "csv", "json", "xlsx"],
+            default="",
+            type=str,
+            dest="export",
+            help="Export plot to png,jpg,pdf,svg file or export dataframe to csv,json,xlsx",
+        )
+        try:
+            ns_parser = parse_known_args_and_warn(parser, other_args)
+            if not ns_parser:
+                return
 
-    def call_sell_house(self, other_args: List[str]):
-        """Process sell_house command"""
-        quiverquant_view.sell_government(other_args, "house")
+            quiverquant_view.display_top_lobbying(
+                num=ns_parser.num, raw=ns_parser.raw, export=ns_parser.export
+            )
+        except Exception as e:
+            print(e, "\n")
 
-    def call_raw_house(self, other_args: List[str]):
-        """Process raw_house command"""
-        quiverquant_view.raw_government(other_args, self.ticker, "house")
+    def _check_ticker(self):
+        """Check if ticker loaded"""
+        if self.ticker:
+            return True
+        print("N ticker loaded.  Use `load <ticker>` first")
```

Comment:
No*
There's an additional extra space after "loaded."
Missing new line

This also applies for the other functions